migrations/accounts.js

```diff
@@ -0,0 +1,39 @@
+// Maps roles to accounts on live network
+const ACCESS_CONTROLLER = "0xaa11C97Be40Cdf6e24229EAA731EE3701C3B9493";
+const LOCKED_ACCOUNT_ADMIN = "0xE8Bba2F6bcF6893E66F316d7f2546ee8534F2868";
+const WHITELIST_ADMIN = "0x45FE1297E62c57179BEbe2cdcBcf22b9005769b4";
+const PLATFORM_OPERATOR_WALLET = "0xfE9641B0f229a2D55a3D1aBC77eCa90648BD5357";
+const PLATFORM_OPERATOR_REPRESENTATIVE =
+  "0x11CA6fdC4117feb52d8b81adA7AdEBd519dD8D08";
+const EURT_DEPOSIT_MANAGER = "0x113b51BFAcbF6BC25B553E2a0B1C90741c25C6eB";
+
+export default function controlAccounts(network, accounts) {
```

Comment:
I think it would be much wiser to return object here. Then calling it and destructing it is a way more readable and self-describing.

Reply:
done!